_includes/cloud-db/cloud-db-backup.md

```diff
@@ -0,0 +1 @@
+All of your databases are automatically backed up daily. FeatureBase handles and manages these backups for you. In the event an issue occurs and you need to restore your data from a backup, or if you have questions or particular needs for backups, please contact [se@featurebase.com](mailto:se@featurebase.com).
```

Comment:
Some things here:
* As far as I'm aware, the discord channel is now the main avenue for raising issues, talking with support
* I'd say we can merge the first two sentences along the following lines:

```
FeatureBase automatically backs-up your databases every 24 hours.

* [Contact FeatureBase Support on Discord](https://discord.gg/featurefirstai){:target="_blank"} to ask questions or to get help restoring a backup.
```